model/Manager.php

```diff
@@ -2,33 +2,15 @@
 
 namespace EmilieSchott\BlogPHP\Model;
 
+use EmilieSchott\BlogPHP\Paginator\Paginator;
 use EmilieSchott\BlogPHP\Model\PDOFactory;
 
-abstract class Manager {
+abstract class Manager extends Paginator {
```

Comment:
Why extend if you create a paginator inside the construct?

Reply:
Hm... I assume this is an oversight of one of my previous attempts to organize my code. I'm removing it.